common/source/docs/common-hv-pm.rst

```diff
@@ -0,0 +1,29 @@
+.. _common-hv-pm:
+
+================================
+HV PM(High voltage power module)
+================================
+
+Overview：
+=========
+
+The CUAV HV_PM module is a new high voltage voltage power module independently developed by CUAV with the following features:
+* Higher voltage input: 10v-60v (3s~14s battery)
+* More accurate battery monitor: 
+  * voltage detection accuracy: +-0.1v
+  * current detection accuracy: +-0.4A
+* Bec (5v) max current: 5A
+* Max (detection) current: 60A
+* Max output current(ESC/MOTOR PORT): 60A
+
+Enable HV PM
+============
+
+.. image:: ../../../images/hv_pm.png
```

Comment:
this image link is not correctly formatted, it needs a target

Reply:
Yes, but its name is can pmu

Reply:
just fyi, actually I dont think a :target: is required....its the images clickable hotlink...if not there, its just an embedded image....almost all the ones existing are merely links to themselves and provide no new info...unlike say a partners logo which hyperlinks to the partners website....